docs/OESDLBugBar.md

```diff
@@ -0,0 +1,86 @@
+# Open Enclave SDL Bug Bar
+
+The information listed in this “bug bar” is intended to help Open Enclave SDK developers to triage bugs and determine the bug severity in terms of security. 
+
+This bug bar and the ratings are derived based on [Microsoft SDL Bug bar](https://aka.ms/sdlbugbar) to apply for the enclaves, specifically to the enclaves created using Intel SGX technology. It is highly recommended to fix all known bugs with critical, important, or moderate severity prior to release. 
```

Comment:
I thought about having this severity pivot instead vulnerability pivot. I feel it is easier to navigate by knowing the class of vulnerability and reaching to the severity than the other way, but it could be just me. If you insist I can change it to severity pivot or even have both.

Reply:
Since https://aka.ms/sdlbugbar prompts for a login, please delete the reference.

Reply:
This didn't require login when I initially added the link, it looks like it is changed now. Anyways I have changed the initial paragraph in my next updates, waiting for reply to some of my questions before pushing my changes.